e2e/tests/transfer/forwarding_test.go

```diff
@@ -91,3 +89,78 @@ func (s *TransferForwardingTestSuite) TestThreeChainSetup() {
 		s.Require().Equal(expected, actualBalance.Int64())
 	})
 }
+
+func (s *TransferForwardingTestSuite) TestForwarding_WithLastChainBeingICS20v1_Succeeds() {
+	ctx := context.TODO()
+	t := s.T()
+
+	threeChainSetup := testsuite.ThreeChainSetup()
+	chainCChainID := "chainC-1" // Setting manually to ensure chainID is set to chainC-1
```

Comment:
I want opinion on this: The other option I considered for this was to export the chain C chain ID from testsuite (the one that `ThreeChainSetup` uses), but it felt a bit weird because we have `func (tc TestConfig) GetChainAID() string` and `func (tc TestConfig) GetChainBID() string` in testconfig.go, and I wasn't sure how to make that smooth. Any ideas are welcome, and I can make this a cleaner, more reusable approach.